README.md

```diff
@@ -46,7 +46,17 @@ We will focus on building an open API first. Then developers can use the API to
 
 ## User stories so far
 
-Our goal is to keep things simple and not reinvent wheels. So far we have only two user roles: participants and chapter organizers.
+Our goal is to keep things simple and not reinvent wheels. So far we have only two user roles: participants and chapter organizers, both of which are users.
+
+### As a user
+
+- I can open a registration page where I can sign up with email password.
+
+- I can log in with my email/password or I can log in with social login via Google.
+
+- I can see my account page where I can reset my password if I've registered with email/password. Otherwise I'll see a link to my Google profile.
```

Comment:
@matjack1 `email/password` sounds like email or password. Please replace `email/password` with `email and password` to keeps things clear.

Btw thanks for these changes :)

Reply:
changed, you're welcome :)